server/src/weaverbird/backends/pandas_executor/geo_utils.py

```diff
@@ -8,8 +8,12 @@ def __init__(self, msg: str) -> None:
 
 
 def df_to_geodf(df: pd.DataFrame) -> gpd.GeoDataFrame:
```

Comment:
```suggestion
def df_to_geodf(df: pd.DataFrame | gpd.GeoDataFrame) -> gpd.GeoDataFrame:
```
?

Reply:
`gpd.GeoDataFrame` inherits `pd.DataFrame` (which can be annoying, because some method are overriden and change the behaviour slightly) so this is already induced